### DISCUSSION TITLE
Cannot build on Linux Mint

### DISCUSSION THREAD

**Discussion by Sod-Almighty** [mono/sdb#69] [Questions] [answered]
I must be doing something wrong, but I'm new to Mono and can't seem to get sdb to build.

I've tried googling for the error messages, but come up empty. Am I missing some kind of dependency?
```
$ git clone https://github.com/mono/sdb.git
$ cd sdb
$ git submodule update --init --recursive
$ make
```
> cd dep/debugger-libs && nuget restore debugger-libs.sln && msbuild /nologo /property:Configuration=Debug /verbosity:quiet debugger-libs.sln
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
WARNING: Error: SendFailure (Error writing headers)
Unable to find version '0.10.0-beta6' of package 'Mono.Cecil'.
Unable to find version '1.3.2' of package 'Microsoft.CodeAnalysis'.
Unable to find version '1.1.0' of package 'Microsoft.CodeAnalysis.Analyzers'.
Unable to find version '1.3.2' of package 'Microsoft.CodeAnalysis.Common'.
Unable to find version '1.3.2' of package 'Microsoft.CodeAnalysis.CSharp'.
Unable to find version '1.3.2' of package 'Microsoft.CodeAnalysis.CSharp.Workspaces'.
Unable to find version '1.3.2' of package 'Microsoft.CodeAnalysis.VisualBasic'.
Unable to find version '1.3.2' of package 'Microsoft.CodeAnalysis.VisualBasic.Workspaces'.
Unable to find version '1.3.2' of package 'Microsoft.CodeAnalysis.Workspaces.Common'.
Unable to find version '1.0.27' of package 'Microsoft.Composition'.
Unable to find version '1.3.1' of package 'System.Collections.Immutable'.
Unable to find version '1.4.2' of package 'System.Reflection.Metadata'.
Unable to find version '4.3.1' of package 'System.ValueTuple'.
Makefile:178: recipe for target 'bin/ICSharpCode.NRefactory.dll' failed
make: *** [bin/ICSharpCode.NRefactory.dll] Error 1

I'm running Linux Mint 18 64-bit, if that helps.

**Comment by alexrp** [comment]
Does `nuget` work at all on your system?

**Comment by Sod-Almighty** [comment]
Dunno, couldn't say. Because rather than sit on my hands producing nothing for SIX MONTHS since I reported this serious show-stopping bug, I have given up on the joke that is Mono and developed my software without it.

What, did you seriously expect me to just wait half a year getting nothing done? That might be how _you_ operate, but rest assured that is not the norm.

**Comment by alexrp** [comment, marked as answer]
This command line tool is not really supported directly by the Mono team; I made it in my spare time and support for it is on a best-effort basis from anyone who's interested (which is mostly/usually just me). Before you jump to any conclusions about expectations, I've been on a fairly lengthy sick leave (and am no longer with Microsoft), so I quite simply have not had the time to triage any issues on this repository.

I am just recently getting around to catching up to GitHub activity that I've missed in that time. I mean, would you rather I'd closed the issue no questions asked?

Anyway, since you say you've moved on without using Mono, I'll close this issue.
